engine/src/main/java/org/terasology/rendering/dag/stateChanges/DisableDepthMask.java

```diff
@@ -61,10 +61,7 @@ public boolean isTheDefaultInstance() {
 
     @Override
     public boolean equals(Object obj) {
-        if (obj instanceof DisableDepthMask) {
-            return this.enabled == ((DisableDepthMask) obj).isEnabled();
-        }
-        return false;
+        return (obj instanceof DisableDepthMask) && this.enabled == ((DisableDepthMask) obj).isEnabled();
```

Comment:
That's correct @tdgunes, it got flagged by one of the inspections before the commit takes place. I don't particularly like it either, the if block from my perspective was more readable, but it's a small thing and I just went along with the recommendation.